src/less/day-slot.less

```diff
@@ -30,20 +31,25 @@
     min-height: 1em;
   }
 
+  .rbc-time-slot {
+    border-top: 1px solid lighten(@cell-border, 10%);
+  }
 }
 
 .rbc-time-slot {
   flex: 1 0 0;
-  min-height: 20px;
-  height: 20px;
 
-  & + & {
-    border-top: 1px solid @cell-border;
+  &.rbc-now {
+    font-weight: bold;
   }
+}
 
-  &:nth-child(even){
-    border-top-color: lighten(@cell-border, 10%);
-  }
+.rbc-timeslice-group {
+  border-bottom: 1px solid @cell-border;
+  min-height: 40px;
+  height: 40px;
```

Comment:
What do you envision the behavior of the slice to be? seems like there are two options, 

1. what is there now which is min heights on the Group which shrinks slices proportionally
2. height on the slice which makes them scale bit better, but groups expand in height without bound.

I am not sure which makes the most sense honestly. It does seem like setting 20px min height on the slices and let the group have whatever height would be a good default as well. it sort of depends on how you envision the slices.